packages/stores/types.ts

```diff
@@ -288,11 +292,21 @@ type TTradersHubStore = {
     selectRegion: (region: string) => void;
     is_real_wallets_upgrade_on: boolean;
     toggleWalletsUpgrade: (value: boolean) => void;
+    openFailedVerificationModal: (selected_account_type: Record<string, any>) => void;
     financial_restricted_countries: boolean;
     selected_account_type: string;
     no_CR_account: boolean;
     no_MF_account: boolean;
-    setTogglePlatformType: () => void;
+    setTogglePlatformType: (value: string) => void;
+    is_tour_open: boolean;
+    selected_platform_type: string;
+    // TODO: must be BrandConfig type but can't import it from appstore. Maybe we need to create the same type in 'stores' package. Have to think about this!
+    available_platforms: any[];
+    CFDs_restricted_countries: boolean;
+    is_demo_low_risk: boolean;
+    selected_region: 'Non-EU' | 'EU' | 'All';
+    getExistingAccounts: (platform: string, market_type: string) => Record<string, any>[];
+    available_dxtrade_accounts: Record<string, any>[];
```

Comment:
@sergei-deriv Many `any`s have been added 😬 Can you please double-check the added types, I believe some of them are already in the master with the correct type.

Reply:
@farzin-deriv fully agree with you, but it was just for mock TS errors and move card to review to define more errors :) I will use this code review time to find/create types, of course.